modules/ocf_desktop/files/xsession/lab_close_notify

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import datetime
+import sched
+import subprocess
+import time
+
+from ocflib.lab import hours
+
+# 0 minutes, 2 minute, 5 minutes, 10 minutes, 15 minutes
+WARNING_DURATIONS = [0, 120, 300, 600, 900]
+
+def notify_user(seconds_left):
+    title = 'Lab closure'
+    if seconds_left != 0:
+        msg = 'Lab closing in {0:g} minutes'.format(seconds_left / 60)
+    else:
+        msg = 'Lab is closing NOW!'
+
+    # TODO: pick an icon
+    subprocess.call(['notify-send', title, msg])
+
+today = datetime.datetime.today()
+todays_hours = hours.REGULAR_HOURS[(today.weekday() - 1) % 7]
+closing_hours = [x.close for x in todays_hours]
+
+# unix timestamps (seconds since epoch) for the closing times today
+closing_times = [datetime.datetime.combine(today.date(), x).timestamp()
+        for x in closing_hours]
```

Comment:
I usually like to split up long lines into shorter ones since it tends to make the code clearer and self-documenting-- but since this makes the line more concise anyways I'll go with your suggestion. Thanks!